metadata.json

```diff
@@ -15,6 +15,10 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 4.13.0 < 6.0.0"
+    },
+    {
+      "name": "puppetlabs/apt",
+      "version_requirement": ">= 6.3.0"
```

Comment:
we need an upper version boundary here as well

Reply:
Ok, may the Documentation be with me :)